Cannot find type `_Pred` in this scope

Hey there.

Currently trying to build out what's on master. Using a brew installed version of llvm for it with all the path variables set. Having a bit of trouble getting the bindings to build though. Full error log is

```
error[E0412]: cannot find type `_Pred` in this scope
   --> skia-bindings/src/bindings.rs:168:25
    |
167 | pub struct std_tuple__EnableUTypesCtor {
    |                                       - help: you might be missing a type parameter: `<_Pred>`
168 |     pub _base: std__And<_Pred>,
    |                         ^^^^^ not found in this scope

error[E0412]: cannot find type `_Pred` in this scope
   --> skia-bindings/src/bindings.rs:168:25
    |
168 |     pub _base: std__And<_Pred>,
    |                         ^^^^^ not found in this scope

error[E0412]: cannot find type `_Pred` in this scope
   --> skia-bindings/src/bindings.rs:173:25
    |
172 | pub struct std_tuple__EnableCopyFromOtherTuple {
    |                                               - help: you might be missing a type parameter: `<_Pred>`
173 |     pub _base: std__And<_Pred>,
    |                         ^^^^^ not found in this scope

error[E0412]: cannot find type `_Pred` in this scope
   --> skia-bindings/src/bindings.rs:173:25
    |
173 |     pub _base: std__And<_Pred>,
    |                         ^^^^^ not found in this scope

error[E0412]: cannot find type `_Pred` in this scope
   --> skia-bindings/src/bindings.rs:178:25
    |
177 | pub struct std_tuple__EnableMoveFromOtherTuple {
    |                                               - help: you might be missing a type parameter: `<_Pred>`
178 |     pub _base: std__And<_Pred>,
    |                         ^^^^^ not found in this scope

error[E0412]: cannot find type `_Pred` in this scope
   --> skia-bindings/src/bindings.rs:178:25
    |
178 |     pub _base: std__And<_Pred>,
    |                         ^^^^^ not found in this scope

error[E0412]: cannot find type `_Pred` in this scope
   --> skia-bindings/src/bindings.rs:183:25
    |
182 | pub struct std_tuple__EnableImplicitCopyFromPair {
    |                                                 - help: you might be missing a type parameter: `<_Pred>`
183 |     pub _base: std__And<_Pred>,
    |                         ^^^^^ not found in this scope

error[E0412]: cannot find type `_Pred` in this scope
   --> skia-bindings/src/bindings.rs:183:25
    |
183 |     pub _base: std__And<_Pred>,
    |                         ^^^^^ not found in this scope

error[E0412]: cannot find type `_Pred` in this scope
   --> skia-bindings/src/bindings.rs:188:25
    |
187 | pub struct std_tuple__EnableExplicitCopyFromPair {
    |                                                 - help: you might be missing a type parameter: `<_Pred>`
188 |     pub _base: std__And<_Pred>,
    |                         ^^^^^ not found in this scope

error[E0412]: cannot find type `_Pred` in this scope
   --> skia-bindings/src/bindings.rs:188:25
    |
188 |     pub _base: std__And<_Pred>,
    |                         ^^^^^ not found in this scope

error[E0412]: cannot find type `_Pred` in this scope
   --> skia-bindings/src/bindings.rs:193:25
    |
192 | pub struct std_tuple__EnableImplicitMoveFromPair {
    |                                                 - help: you might be missing a type parameter: `<_Pred>`
193 |     pub _base: std__And<_Pred>,
    |                         ^^^^^ not found in this scope

error[E0412]: cannot find type `_Pred` in this scope
   --> skia-bindings/src/bindings.rs:193:25
    |
193 |     pub _base: std__And<_Pred>,
    |                         ^^^^^ not found in this scope

error[E0412]: cannot find type `_Pred` in this scope
   --> skia-bindings/src/bindings.rs:198:25
    |
197 | pub struct std_tuple__EnableExplicitMoveFromPair {
    |                                                 - help: you might be missing a type parameter: `<_Pred>`
198 |     pub _base: std__And<_Pred>,
    |                         ^^^^^ not found in this scope

error[E0412]: cannot find type `_Pred` in this scope
   --> skia-bindings/src/bindings.rs:198:25
    |
198 |     pub _base: std__And<_Pred>,
    |                         ^^^^^ not found in this scope

For more information about this error, try `rustc --explain E0412`.
```

Wondering if someone's hit this before. I've tried looking through the repo for any definition to `_Pred` but have not been able to find any.

--- a/skia-bindings/build_support/skia_bindgen.rs
+++ b/skia-bindings/build_support/skia_bindgen.rs
@@ -409,6 +409,8 @@ const OPAQUE_TYPES: &[&str] = &[
     "GrD3DMemoryAllocator",
     // m87, yuva_pixmaps
     "std::tuple",
+    // Homebrew macOS LLVM 13
+    "std::tuple_.*",
     // m93: private, exposed by Paint::asBlendMode(), fails layout tests.
     "skstd::optional",
 ];

